pkg/controller/kieapp/kieapp_controller.go

```diff
@@ -198,7 +197,7 @@ func (reconciler *ReconcileKieApp) Reconcile(request reconcile.Request) (reconci
 		return reconcile.Result{}, err
 	}
 
-	log := log.WithValues("kind", instance.Kind, "name", instance.Name, "namespace", instance.Namespace)
+	log := log.With("kind", instance.Kind, "name", instance.Name, "namespace", instance.Namespace)
```

Comment:
This syntax (or the previous one, WithValues) seems to be one distinguishing factor of the logger. That means adopting it in our layer would bind us to this logger. Question is, how useful is it? I see the log var only being used once below, meaning the same info could have been provided down there. What are your thoughts?

Reply:
i find it very useful... but give it a spin and offer your thoughts

Reply:
Sounds good... I can imagine it being useful, but I assume only if you create a logger with the properties and then log with it several times.

Let me know about the probe before I merge this PR.

Reply:
true, ill look at whether we are better off passing in the args as needed

Reply:
I meant about the liveness probe! :-)